pegasus/sites.v3/code.org/views/analytics_event_log_helper.haml

```diff
@@ -0,0 +1,3 @@
+%script{src: webpack_asset_path('js/code.org/views/analytics_event_log_helper.js')}
+
+#analytics-event-log-helper{"data-event-name" => event_name}
```

Comment:
nice partial! this looks like it will be relatively easy to update to start using the ruby SDK once it becomes available.